jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/api/util/InitContainerHandlerTest.java

```diff
@@ -23,109 +23,106 @@
     }
 
     @Test
     public void removeAll() {
         PodTemplateSpecBuilder builder = getPodTemplateBuilder("bla", "foo/bla");
-        assertTrue(handler.hasInitContainer(builder, "bla"));
+        assertThat(handler.hasInitContainer(builder, "bla")).isTrue();
         handler.removeInitContainer(builder, "bla");
-        assertFalse(handler.hasInitContainer(builder, "bla"));
+        assertThat(handler.hasInitContainer(builder, "bla")).isFalse();
         verifyBuilder(builder, null);
     }
 
     @Test
     public void removeOne() {
         PodTemplateSpecBuilder builder = getPodTemplateBuilder("bla", "foo/bla", "blub", "foo/blub");
-        assertTrue(handler.hasInitContainer(builder, "bla"));
-        assertTrue(handler.hasInitContainer(builder, "blub"));
+        assertThat(handler.hasInitContainer(builder, "bla")).isTrue();
+        assertThat(handler.hasInitContainer(builder, "blub")).isTrue();
         handler.removeInitContainer(builder, "bla");
-        assertFalse(handler.hasInitContainer(builder, "bla"));
-        assertTrue(handler.hasInitContainer(builder, "blub"));
+        assertThat(handler.hasInitContainer(builder, "bla")).isFalse();
+        assertThat(handler.hasInitContainer(builder, "blub")).isTrue();
         verifyBuilder(builder, Collections.singletonList(createInitContainer("blub", "foo/blub")));
     }
 
     @Test
     public void existingSame() {
-        new Expectations() {{
-            log.warn(anyString, withSubstring("blub"));
-        }};
-
+        doNothing().when(log).warn(anyString(), anyString());
         PodTemplateSpecBuilder builder = getPodTemplateBuilder("blub", "foo/blub");
-        assertTrue(handler.hasInitContainer(builder, "blub"));
+        assertThat(handler.hasInitContainer(builder, "blub")).isTrue();
         Container initContainer = createInitContainer("blub", "foo/blub");
         handler.appendInitContainer(builder, initContainer);
-        assertTrue(handler.hasInitContainer(builder, "blub"));
+        assertThat(handler.hasInitContainer(builder, "blub")).isTrue();
         verifyBuilder(builder, Collections.singletonList(initContainer));
     }
 
     @Test
     public void existingDifferent() {
         try {
             PodTemplateSpecBuilder builder = getPodTemplateBuilder("blub", "foo/bla");
-            assertTrue(handler.hasInitContainer(builder, "blub"));
+            assertThat(handler.hasInitContainer(builder, "blub")).isTrue();
             Container initContainer = createInitContainer("blub", "foo/blub");
-            handler.appendInitContainer(builder, initContainer);
-            fail();
+            assertThatIllegalArgumentException()
+                    .isThrownBy(() -> handler.appendInitContainer(builder, initContainer));
+
         } catch (IllegalArgumentException exp) {
-            assertTrue(exp.getMessage().contains("blub"));
+            assertThat(exp.getMessage()).contains("blub");
         }
```

Comment:
With `assertThatIllegalArgumentException` this try-catch block should not be needed